diagnostic_aggregator/CHANGELOG.rst

```diff
@@ -2,6 +2,28 @@
 Changelog for package diagnostic_aggregator
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Forthcoming
```

Comment:
I'd remove the changelog additions. It looks like the maintainer usually does that all at once at release rather than in PRs: https://github.com/ros/diagnostics/commit/13bae1a6caaa721d53235eb19555b69f427995e8